### PR TITLE
API for CVP calls

### PR DESCRIPTION
**Pull request by gbonnoron**
As CVP is not longer experimental in fplll, we can start to use it from fpylll.

### CODE REVIEW

File: src/fpylll/fplll/svpcvp.pyx [hunk 1]
```diff
@@ -89,7 +90,46 @@ class SVP:
...
     OVERRIDE_BND = SVP_OVERRIDE_BND
 
-# class CVP:
-#     closest_vector = closest_vector
-#     DEFAULT = CVP_DEFAULT
-#     VERBOSE = CVP_VERBOSE
+def closest_vector(IntegerMatrix B, target, int flags=CVP_DEFAULT):
+    """Return a closest vector.
+
+    :param IntegerMatrix B:
```
Comment (malb) [inline src/fpylll/fplll/svpcvp.pyx]: Can I be that guy and ask for more documentation here?

**Comment by malb** [comment]
Any chance, I can convince you to add tests, e.g. a doctest?

**Comment by malb** [comment]
Thanks. This doesn't really depend on the PR on the fplll side being already merged, right? So we could merge it once the doctest failure is resolved?

**Comment by gbonnoron** [comment]
Now it is test_numpy.py that fail the build. Any idea why?

**Comment by malb** [comment]
Mhhh, when I got those in the past it usually indicated a problem elsewhere (?) You could setup a virtualenv for Python 3 and try it locally? Sorry, not very helpful, I know.

**Comment by malb** [comment]
Looks like it was a glitch in the matrix.

**Pull request merged**